uniffi_bindgen/src/bindings/python/gen_python/mod.rs

```diff
@@ -24,14 +25,20 @@ mod miscellany;
 mod object;
 mod primitives;
 mod record;
-mod wrapped;
 
-// Some config options for it the caller wants to customize the generated python.
-// Note that this can only be used to control details of the python *that do not affect the underlying component*,
-// sine the details of the underlying component are entirely determined by the `ComponentInterface`.
+// Config options to customize the generated python.
 #[derive(Debug, Clone, Default, Serialize, Deserialize)]
 pub struct Config {
     cdylib_name: Option<String>,
+    #[serde(default)]
+    custom_types: HashMap<String, CustomTypeConfig>,
+}
+
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+pub struct CustomTypeConfig {
+    imports: Option<Vec<String>>,
+    into_custom: TemplateExpression,
+    from_custom: TemplateExpression,
```

Comment:
I _think_ we might want to move this to a more common spot once we get land this.

Reply:
I was wondering about that.  The only reason not to is that the fields vary slightly between different languages (python doesn't have `type_name`, on kotlin/swift it's optional).  That's not all that compelling though, I could go either way on this one.

Reply:
Hmm, yes. Worth a comment perhaps?